k8s/jobs/kustomize/jobs/overlays/common/issuer/register-tenant.sh

```diff
@@ -7,9 +7,11 @@
 echo "Adding curl and jq"
 apk --no-cache add curl jq
 
+ARCH=$( uname -m | sed 's/x86_64/amd64/')
```

Comment:
Need to add this here

UNAME=$( uname -m)
case $UNAME in
x86_64)
ARCH="amd64"
;;
aarch64)
ARCH="arm64"
;;
*)
ARCH="amd64"
;;
esac

Reply:
Done. Note that I removed the `*)` case as it causes `arm64` to get incorrectly converted to `amd64`.